blog/content/post/2022-01-17-kgpe_D16_status.md

```diff
@@ -0,0 +1,149 @@
+The same was done for OpenBMC
+https://www.raptorengineering.com/coreboot/kgpe-d16-bmc-port-offer.php
+in the same year. The initial source code is still available at
+[Raptor Engineering's site](https://www.raptorengineering.com/coreboot/kgpe-d16-bmc-port-status.php).
+
+The platform support lived on the main coreboot branch for 3 years until the
+[4.12 release](https://doc.coreboot.org/releases/coreboot-4.12-relnotes.html)
+where features like C bootblock, postcar phase and relocatable ramstage support
+became mandatory. The board support has been dropped from the main branch due
+to insufficient interest and maintainership of the platform. Additionally the
+code upstreamed was not the best quality, most likely because Raptor did not
+gather the required funds to clean up and improve the code. It has been merged
+as is with insufficient review which led to a code drop few years later.
+
+# To the rescue of KGPE-D16
+
+It has been a huge blow for the community believing in privacy and liberty of
+the hardware. Thus 3mdeb tried to answer on the community requests and needs to
+bring back the platform. To prevent the board from dropping (half a year before
+the 4.12 release at the end of 2019, where the deprecation has been announced)
+3mdeb has applied for funding to [NLnet Foundation](https://nlnet.nl/),
+unfortunately the project to improve the quality of board support has been
+rejected. The application can be found on [3mdeb's GitHub](https://github.com/3mdeb/kgpe-osf/blob/master/docs/nlnet-application.md)
+
+The hope was almost lost. In the meantime 3mdeb propagated the need to protect
+the libre hardware like KGPE-D16 on various events like, e.g. FOSDEM:
+
+* [FOSDEM'20](https://archive.fosdem.org/2020/schedule/event/coreboot_amd/)
+* [FOSDEM'21](https://archive.fosdem.org/2021/schedule/event/firmware_osfsoap2/)
+
+![](/img/fosdem_logo.png)
+
+And fortunately our screams have been heard... [Vikings](Vikings.net) donated
+two KGPE-D16 boards for the development and [Immunefi](https://immunefi.com/)
+has offered to sponsor the effort of bringing back the KGPE-D16 to coreboot
+main tree. For the details visit [heads issue on GitHub](https://github.com/osresearch/heads/issues/719).
+
+# KGPE-D16 revival
+
+3mdeb started the code refactoring and improving efforts in September 2021. But
+the scope was not only to bring the code quality to coreboot's current
+standards, no. We (3mdeb and the sponsors) aimed to create even more secure and
+libre platform than it was before (back then in main coreboot branch). Besides
+the coreboot requirements 3mdeb has enabled and validated TPM 2.0 and vboot on
+the platform using 8 MB and 16 MB SPI flashes (much larger than 2 MB default
+ones). 3 months later the finalized firmware has been released under
+[Dasharo](https://dasharo.com/) brand with multi-flavored pre-built binaries
+ready to use by end users. Now the platform benefits from automated tests,
```

Comment:
and transparen validation, when test results would be published

Reply:
Added